templates/zerver/reset_confirm.html

```diff
@@ -63,7 +65,7 @@ <h1>{{ _('Set a new password.') }}</h1>
 
                 <div class="input-box m-t-30">
                     <div class="centered-button">
-                        <button type="submit" class="" value="Submit">Submit</button>
+                        <button type="submit" class="register-button" value="Submit">Submit</button>
```

Comment:
`register-button` does not seem the best class name here. Would be good to use some other name like `submit-button`, but then the CSS selector should have use some unique class or ID as `submit-button` class might be used in other places.

Reply:
Hey @sahil839,  

I had previously done the same by assigning unique IDs to both buttons as `signup-button` and `reset-button` and using these ID selectors to define the CSS. However, you suggested [here](https://github.com/zulip/zulip/pull/32141#discussion_r1853777966) removing those new `id`s and using the existing `register-button` selector to define the disabled properties.  

Should I revert to the previous version where I defined new IDs for both buttons? 

Thanks!

Reply:
I was thinking of adding a common class to both buttons and use selector like `.password-reset .submit-button` and similarly for signup page as well, but using IDs is also fine.